modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/processor/OperationMessageProcessor.java

```diff
@@ -129,6 +130,8 @@ public void initialise() throws InitialisationException
     {
         returnDelegate = createReturnDelegate();
         operationExecutor = operationModel.getExecutor().createExecutor();
+        ConnectionManagerAdapter connectionManager = new DefaultConnectionManager(muleContext);
```

Comment:
Inject it, don't create one yourself. Otherwise you end up having one of this per insance and tap will never work